bin/install.py

```diff
@@ -126,3 +130,6 @@ def makedir(directory):
 
 download_and_install(BASH_COMPLETION_URL, BASH_COMPLETION_TARGET)
 print("Installed Bash completion in '%s'" % BASH_COMPLETION_TARGET)
+
+download_and_install(SYSTEMD_SERVICE_URL, SYSTEMD_SERVICE_TARGET)
```

Comment:
Can we do this if it's only Linux/BSD?

Reply:
Is there any reason to do this only for Linux/BSD? Bash and zsh completions are downloaded even if you don't have bash/zsh installed.